var/sabath/assets/sabath/models/c/cosmoflow/cosmoflow.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/configs/cosmo.yaml b/configs/cosmo.yaml
+index 675e32f..d164966 100644
+--- a/configs/cosmo.yaml
++++ b/configs/cosmo.yaml
+@@ -8,10 +8,10 @@ mlperf:
+ 
+ data:
+     name: cosmo
+-    data_dir: /global/cscratch1/sd/sfarrell/cosmoflow-benchmark/data/cosmoUniverse_2019_05_4parE_tf_v2
++    data_dir: /path/to/cosmoUniverse_2019_05_4parE_tf_v2_mini/ #Update path according where your dataset is located
```

Comment:
Where is the dataset located? The dataset was installed with the `fetch` command so this is an internal decision of the project

Reply:
The dataset should be located at "$SABATH_ROOT/var/sabath/cache", should I update data_dir in the patch accordingly?

Reply:
This sounds like a perfect recipe for a piece of code that goes into SABATH. The input file or its patch could become a template and have Python's `string.Template` do the work of creating correct patch on the fly.